source/events/pointing.rst

```diff
@@ -0,0 +1,56 @@
+.. include:: ../references.txt
+
+.. _iact-pnt:
+
+.. warning::
+   This is a first draft proposal of a pointing table. Please note that
+   the format is likely subject to change.
+
+
+``POINTING`` extension
+======================
+
+The ``POINTING`` extension is a binary FITS table that contains for a number
+of time stamps the pointing direction of the telescopes. A *pointing* is here
+defined as the centre of the field of view (or centre of the camera
+coordinates). In reality, all telescopes may point to different positions
+(for example for divergent pointing mode).
```

Comment:
I like your POINTING table suggestion and have no significant comments.
Here in the intro you could mention that the main purpose of POINTING at a given time is to serve as a parameter used in IRF access (i.e. to compute exposure or background).